SUIApp/SUIApp/View/ClimateView.swift

```diff
@@ -0,0 +1,386 @@
+            RoundedRectangle(cornerRadius: 20)
+                .fill(.regularMaterial.opacity(0.95))
+        )
+        .ignoresSafeArea(.all, edges: .bottom)
+        .offset(y: UIScreen.main.bounds.height)
+        .offset(y: climateViewModel.currentMenuOffsetY)
+        .gesture(dragGesture)
+    }
+
+    private var visibaleBottomStackView: some View {
+        HStack {
+            VStack(alignment: .leading, spacing: 10) {
+                Text(climateViewModel.updateACTitle)
+                    .font(.system(size: 24))
+                    .bold()
+                Text(Constants.descriptionText)
+                    .foregroundColor(.gray)
+                    .font(.subheadline)
+            }
+            .frame(width: 200)
+            Spacer()
+            Button {
+                climateViewModel.resetSetting()
+            } label: {
+                Image(systemName: Constants.powerImageSystemName)
+                    .resizable()
+                    .frame(width: 50, height: 50)
+            }
+        }
+    }
+
+    private var hiddenBottomSheetStackView: some View {
+        HStack {
+            colorPickerView
+            Spacer()
+            stepperStackView
+                .padding(.bottom, 60)
+            Spacer()
+            ventStackView
+        }
+        .padding(.horizontal, 30)
+        .padding(.top)
+    }
+
+    private var colorPickerView: some View {
+        VStack(spacing: 40) {
+            ColorPicker(Constants.emptyText, selection: $climateViewModel.currentColor)
+                .frame(width: 10)
+            Text(Constants.colorText)
+        }
+    }
+
+    private var ventStackView: some View {
+        VStack(spacing: 40) {
+            Image(Constants.carDoorImageName)
+            Text(Constants.ventText)
+        }
+    }
+
+    private var stepperStackView: some View {
+        VStack(spacing: 40) {
+            HStack(spacing: 40) {
+                Button {
+                    withAnimation {
+                        climateViewModel.decrementAC()
+                    }
+                } label: {
+                    Image(systemName: Constants.chevronLeftImageSystemName)
+                        .resizable()
+                        .frame(width: 10, height: 12)
+                }
+                Text("\(Int(climateViewModel.acSlider))\(Constants.degreeText)")
+                    .font(.system(size: 40))
+                    .fontWeight(.medium)
+                Button {
+                    withAnimation {
+                        climateViewModel.incrementAC()
+                    }
+                } label: {
+                    Image(systemName: Constants.chevronRightImageSystemName)
+                        .resizable()
+                        .frame(width: 10, height: 12)
+                }
+            }
+            .foregroundColor(.white)
+        }
+        .disabled(!climateViewModel.isStart)
+    }
+
+    private var customAlertView: some View {
```

Comment:
```suggestion
    private var customAlertView: some View {
```
вынеси во вью отдельную